src/WidgetBase.ts

```diff
@@ -78,7 +78,7 @@ export class WidgetBase<P = WidgetProperties, C extends DNode = DNode> implement
 	/**
 	 * Array of property keys considered changed from the previous set properties
 	 */
-	private _changedPropertyKeys: string[] = [];
+	private _changedPropertyKeys: (string | number)[] = [];
```

Comment:
This should just be `string`, it is the actual key of the property on the properties object